client/src/pages/Register.tsx

```diff
@@ -40,27 +45,42 @@ export function Register() {
   }
 
   return (
-    <div className="px-[15px] big:px-[50px]">
+    <div className="px-[15px] big:px-[50px] max-w-[600px]">
       <h1 className="text-[24px] my-3">Register</h1>
       <form onSubmit={handleSubmit}>
         <div className="flex flex-col space-y-2">
-          <label>Username</label>
+          <label htmlFor="reg-user-input">Username</label>
```

Comment:
You could also wrap the `label` around the `input`. Then you wouldn't need the `id`.